test/AppNotificationTests/BaseTestSuite.cpp

```diff
@@ -598,3 +598,19 @@ void BaseTestSuite::VerifyExplicitAppId()
     RegisterWithAppNotificationManager();
     AppNotificationManager::Default().Unregister();
 }
+
+void BaseTestSuite::VerifyToastDevicesData()
```

Comment:
Check for a test case with only Microphone and Speaker.
If user doesn't provide any one or two of three ids then default should be empty string / some string that says system default sent to framework udk.

Reply:
More test cases can be added

Reply:
will add tests on those lines